File: AvitoStatistcService_Java/src/main/java/ru/espada/avito/statistic/model/request/GetStatisticAboutCategoryInLocationInPeriodRequest.java
package ru.espada.avito.statistic.model.request;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
// Запрос на получение предложения об изменении цены на продукт в локации в зависимости от времени
public class GetStatisticAboutCategoryInLocationInPeriodRequest {

    private long categoryId;
    private long locationId;
    private long dateStart;
    private long dateEnd;

}

File: UserApiService_C#/AdminApiService/Models/Requests/GetStatisticAboutCategoryInLocationInPeriodRequest.cs
using Newtonsoft.Json;

namespace AdminApiService.Models.Requests;

public class GetStatisticAboutCategoryInLocationInPeriodRequest
{
    [JsonProperty("categoryId")]
    public long categoryId { get; set; }
    [JsonProperty("locationId")]
    public long locationId { get; set; }
    [JsonProperty("dateStart")]
    public long dateStart { get; set; }
    [JsonProperty("dateEnd")]
    public long dateEnd { get; set; }
}